### PR TITLE
README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vishjain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,48 +8,26 @@ This plugin allows Grafana users to use Pixie as a datasource in their Grafana d
...
-### Building from source: front-end
+### Installing the Plugin on an Existing Grafana with the CLI
 
-1. Install dependencies
+Grafana comes with a command line tool that can be used to install plugins.
 
-   ```bash
-   yarn install
-   ```
+1. Upgrade Grafana to the latest version. Get that [here](https://grafana.com/grafana/download/).
```
Comment (htroisi) [inline README.md:15]: Doesn't Pixie require a certain version of Grafana?

File: README.md [hunk 2]
```diff
@@ -8,48 +8,26 @@ This plugin allows Grafana users to use Pixie as a datasource in their Grafana d
...
+2. Run this command: `grafana-cli plugins install pixie-datasource`
+3. Restart the Grafana server.
```
Comment (htroisi) [inline README.md:17]: Link to https://grafana.com/docs/grafana/latest/installation/restart-grafana/

File: README.md [hunk 3]
```diff
@@ -8,48 +8,26 @@ This plugin allows Grafana users to use Pixie as a datasource in their Grafana d
...
-1. Install dependencies
+Grafana comes with a command line tool that can be used to install plugins.
 
-   ```bash
-   yarn install
-   ```
+1. Upgrade Grafana to the latest version. Get that [here](https://grafana.com/grafana/download/).
+2. Run this command: `grafana-cli plugins install pixie-datasource`
+3. Restart the Grafana server. 
+4. In Grafana, Add Datasources and Add Pixie Datasource Plugin.
```
Comment (htroisi) [inline README.md:18]: With Grafana open, from the left vertical menu choose "Configuration" > "Data Sources." Select the "Add data source" button. Search for "Pixie Grafana Datasource Plugin" and press the "Select" button. The Pixie Plugin requires a Pixie API Key and Cluster ID to execute queries. To create an API Key, follow the directions [here](https://docs.pixielabs.ai/using-pixie/api-quick-start/#get-an-api-token). To find your cluster's ID, follow the directions [here](https://docs.pixielabs.ai/using-pixie/api-quick-start/#get-a-cluster-id). Finally, select the "Save & Test" button.

File: README.md (hunk 1 above)
Comment (vishjain) [reply README.md:15]: Ahhh... you're right let me re-word this.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+   yarn watch
+   ```
+
+3. Build plugin in production mode
+
+   ```bash
+   yarn build
+   ```
+
+### Building from source: back-end
```
Comment (htroisi) [inline CONTRIBUTING.md:85]: Do you need to modify the `.ini` file to point to the local repo?

Comment (vishjain) [reply CONTRIBUTING.md:85]: I don't want to go into that deeply in this part. I'll quickly go over this part.

**Review by nserrino — APPROVED** [review]

**Pull request merged**